### PR TITLE
chore: update all ns to chaos-mesh

### LINKED ISSUE #326: Use `chaos-mesh` instead of `chaos-testing` in latest version
After chaos-mesh/chaos-mesh#2818 has been merged, we need to update documents, use `chaos-mesh` instead of `chaos-testing` in latest version.

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

Close #326.

Refer to https://github.com/chaos-mesh/chaos-mesh/issues/2818, after we archive our old versions, the rest versions should be updated from namespace `chaos-testing` to `chaos-mesh` for consistency.

### CODE REVIEW

File: blog/2020-01-15-chaos-mesh-your-chaos-engineering-solution.md [hunk 1]
```diff
@@ -170,14 +170,14 @@ Chaos Mesh runs on Kubernetes v1.12 or later. Helm, a Kubernetes package managem
...
-   helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-testing
+   helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-mesh
    ```
 
    Wait until all components are installed, and check the installation status using:
 
    ```bash
    // check chaos-mesh status
-   kubectl get pods --namespace chaos-testing -l app.kubernetes.io/instance=chaos-mesh
+   kubectl get pods --namespace chaos-mesh -l app.kubernetes.io/instance=chaos-mesh
```
Comment (Oreoxmt) [inline blog/2020-01-15-chaos-mesh-your-chaos-engineering-solution.md:180]: Does the value of `namespace` in the yaml file need to be modified? [L201](https://github.com/chaos-mesh/website/pull/361/files#diff-67d00cd5aa80f91c543ec53115c756ffb06b6bd1959121ed09bb929db2b3c9b0L201)

Comment (g1eny0ung) [reply blog/2020-01-15-chaos-mesh-your-chaos-engineering-solution.md:180]: Yep and good catch! I just searched and replaced all that appeared `chaos-testing`. Seems the value of the namespace in the YAML file wasn't matched...I will try to update them now.

Comment (g1eny0ung) [reply blog/2020-01-15-chaos-mesh-your-chaos-engineering-solution.md:180]: The truth is I forgot to replace `namespace: chaos-testing`. 😑 Will fix it soon.

Comment (g1eny0ung) [reply blog/2020-01-15-chaos-mesh-your-chaos-engineering-solution.md:180]: Done in [e29ca14](https://github.com/chaos-mesh/website/pull/361/commits/e29ca14c306a51c2c54c93182ebb56f48630afa0).

**Comment by STRRL** [comment]
I global searched `chaos-testing`, and found that there are several `chaos-testing` literally existed in these docs:

- versioned_docs/version-2.3.2/quick-start.md
- versioned_docs/version-2.3.2/offline-installation.md
- versioned_docs/version-2.3.2/simulate-jvm-application-chaos.md
- versioned_docs/version-2.3.2/add-new-chaos-experiment-type.md
- versioned_docs/version-2.3.2/production-installation-using-helm.md
- ~~blog/2021-07-07-restrict-authorization.md~~ (I think we do not need update this blog post)
- versioned_docs/version-2.4.1/simulate-jvm-application-chaos.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.3.2/offline-installation.md
- i18n/zh/docusaurus-plugin-content-docs/current/add-new-chaos-experiment-type.md
- versioned_docs/version-2.2.3/simulate-jvm-application-chaos.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.4.1/simulate-jvm-application-chaos.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.3.2/simulate-jvm-application-chaos.md
- i18n/zh/docusaurus-plugin-content-docs/current/simulate-jvm-application-chaos.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.2.3/add-new-chaos-experiment-type.md
- i18n/zh/docusaurus-plugin-content-docs/current/quick-start.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.2.3/quick-start.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.2.3/offline-installation.md
- i18n/zh/docusaurus-plugin-content-docs/current/offline-installation.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.2.3/production-installation-using-helm.md
- i18n/zh/docusaurus-plugin-content-docs/current/production-installation-using-helm.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.4.1/offline-installation.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.4.1/add-new-chaos-experiment-type.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.4.1/quick-start.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.3.2/quick-start.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.4.1/production-installation-using-helm.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.3.2/production-installation-using-helm.md
- i18n/zh/docusaurus-plugin-content-docs/version-2.3.2/add-new-chaos-experiment-type.md
- versioned_docs/version-2.2.3/add-new-chaos-experiment-type.md
- versioned_docs/version-2.2.3/quick-start.md
- versioned_docs/version-2.2.3/offline-installation.md
- versioned_docs/version-2.4.1/add-new-chaos-experiment-type.md
- versioned_docs/version-2.2.3/production-installation-using-helm.md
- versioned_docs/version-2.4.1/quick-start.md
- versioned_docs/version-2.4.1/offline-installation.md
- versioned_docs/version-2.4.1/production-installation-using-helm.md
- docs/simulate-jvm-application-chaos.md
- docs/add-new-chaos-experiment-type.md
- docs/quick-start.md
- docs/offline-installation.md
- docs/production-installation-using-helm.md


Should we also replace them?

**Comment by g1eny0ung** [comment]
@STRRL Yes. We should be. I also forgot to search the single `chaos-testing` word everywhere...

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**